mParticle-Apple-SDK/Data Model/MPConsumerInfo.mm

```diff
@@ -176,39 +176,54 @@ - (NSDictionary *)dictionaryRepresentation {
 @end
 
 #pragma mark - MPConsumerInfo
+@interface MPConsumerInfo ()
+@property (nonatomic, strong, nonnull) dispatch_queue_t queue;
+@end
+
 @implementation MPConsumerInfo
 
 @synthesize cookies = _cookies;
 @synthesize mpId = _mpId;
 @synthesize uniqueIdentifier = _uniqueIdentifier;
 
 - (id)init {
+    // Serial since read/write lock not appropriate given how mpId is created.
+    dispatch_queue_t newQueue = dispatch_queue_create("com.mParticle.MPCookie.queue", DISPATCH_QUEUE_SERIAL);
+    
     self = [super init];
-    if (self) {
+    if (self && newQueue) {
```

Comment:
Why does the initialization of the instance depends on creating a queue? Can't the instance be created and the serial queue be lazily created in the getter of the `queue` property?

```
- (dispatch_queue_t)queue {
    if (!_queue) {
        _queue = dispatch_queue_create("com.mParticle.MPCookie.queue", DISPATCH_QUEUE_SERIAL);
    }
    return _queue;
}
```

Can we also rename the property to a more descriptive name such as `cookieQueue` or `accessQueue`?